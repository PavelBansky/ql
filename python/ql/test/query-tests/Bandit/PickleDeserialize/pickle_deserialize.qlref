Bandit/pickle_deserialize.ql