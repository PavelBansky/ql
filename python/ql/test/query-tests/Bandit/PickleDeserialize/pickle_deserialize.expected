| Test.py:7:8:7:41 | Attribute() | Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue. |
| Test.py:13:7:13:27 | Attribute() | Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue. |
| Test.py:16:7:16:22 | Attribute | Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue. |
| Test.py:20:7:20:31 | Attribute() | Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue. |
| Test.py:28:7:28:23 | Attribute | Pickle and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue. |
